.github/workflows/ci-build.yml

```diff
@@ -61,21 +63,30 @@ jobs:
           java --version
           echo Clojure: $(clojure -M --eval "(clojure-version)")
           clojure --version
-          echo Leiningen: $(lein --version)
 
-      - name: Install Metabase
-        run: ./scripts/install-metabase-jar.sh
-
-      - name: Run tests
-        run: lein test
+      - name: Prepare Metabase deps
+        run: |
+          cd "$GITHUB_WORKSPACE/metabase"
+          source "./bin/prep.sh"
+          prep_deps
 
-      - name: Run linter
+      - name: Build driver
         run: |
-          mkdir -p .clj-kondo/.cache
-          lein lint
+          cd "$GITHUB_WORKSPACE/metabase-driver"
+          clojure -X:test:deps prep
+          clojure -X:dev:build
```

Comment:
I've updated the example you were following in the sudoku driver's [README](https://github.com/metabase/sudoku-driver/) to build the driver. We updated the build script and currently need absolute paths. You can build your driver like
```
clojure -X:build :project-dir "\"$(pwd)\""
```
The quoting of strings from subshells gets a bit wonky but this ensures the `:project-dir` is an absolute path rather than the previous relative version `"."`